protos/sigstore_bundle.proto

```diff
@@ -48,12 +50,12 @@ message TimestampVerificationData {
 // signatures.
 message VerificationMaterial {
         oneof content {
-                dev.sigstore.common.v1.PublicKeyIdentifier public_key = 1;
-                dev.sigstore.common.v1.X509CertificateChain x509_certificate_chain = 2;
+                dev.sigstore.common.v1.PublicKeyIdentifier public_key = 1 [(google.api.field_behavior) = REQUIRED];
+                dev.sigstore.common.v1.X509CertificateChain x509_certificate_chain = 2 [(google.api.field_behavior) = REQUIRED];
         }
         // This is the inclusion promise and/or proof, where
         // the timestamp is coming from the transparency log.
-        repeated dev.sigstore.rekor.v1.TransparencyLogEntry tlog_entries = 3;
+        repeated dev.sigstore.rekor.v1.TransparencyLogEntry tlog_entries = 3 [(google.api.field_behavior) = REQUIRED];
         // Timestamp verification data, over the artifact's signature.
         TimestampVerificationData timestamp_verification_data = 4;
```

Comment:
N.B.: I didn't mark this as required, since I wasn't sure if it was a required part of every Sigstore verification (e.g. `sigstore-python` does not currently do anything with timestamps). But if it should be, let me know 🙂

Reply:
Agreed! You just need a timestamp from somewhere, it could be this or the tlog entry, but it's hard to represent that with required labels

Reply:
Makes sense! Yeah, it sounds like it makes sense to annotate neither, and have the client select one or both (but not none).